app/lib/frontend/models.dart

```diff
@@ -50,6 +50,9 @@ class Package extends db.ExpandoModel {
   @db.ModelKeyProperty(propertyName: 'latest_dev_version')
   db.Key latestDevVersionKey;
 
+  @CompatibleStringListProperty()
```

Comment:
Do we have to use`CompatibleStringListProperty` for new properties?

Reply:
Can we give it a comment `/// List of [User.userId]`

Can we give `uploaderEmails` a comment, that that is deprecated, and should not be modified again?

Reply:
> Do we have to use`CompatibleStringListProperty` for new properties?

Switching to `StringListProperty`.

> Can we give `uploaderEmails` a comment, that that is deprecated, and should not be modified again?

I'll do a follow-up PR that changes both properties in-sync (after this and the PR with the AuthenticatedUser gets merged). After that is deployed, we can remove it altogether.